matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXMegolmDecryption.kt

```diff
@@ -309,9 +249,9 @@ internal class MXMegolmDecryption(
         )
 
         when (addSessionResult) {
-            is MXOlmDevice.AddSessionResult.Imported               -> addSessionResult.ratchetIndex
+            is MXOlmDevice.AddSessionResult.Imported -> addSessionResult.ratchetIndex
             is MXOlmDevice.AddSessionResult.NotImportedHigherIndex -> addSessionResult.newIndex
-            else                                                   -> null
+            else -> null
```

Comment:
this update coming from tchap or element ?

Reply:
it's come from Element, if you talk about the line 254 or 252